include/experimental/group/gemm/compute_policy.hpp

```diff
@@ -89,14 +88,68 @@ struct compute_policy_int4_dequantize<
 
   static constexpr uint32_t block_size_y_a = 16;
   using mma_attr = mma_attr_t<arch_tag_, block_size_y_a>;
-  static constexpr uint32_t block_bytes_x_a =
-      (mma_engine == mma_engine::xmx) ? mma_attr::mma_k_in_bytes : 32;
+  static constexpr uint32_t block_bytes_x_a = mma_attr::mma_k_in_bytes;
+  static constexpr uint32_t block_size_x_a =
+      block_bytes_x_a / sizeof(dtype_mma_a);
+  static constexpr uint32_t block_size_x_b = mma_attr::mma_n_in_elem;
+  static constexpr uint32_t block_bytes_y_b = mma_attr::mma_k_in_bytes;
+  static constexpr uint32_t block_size_y_b =
+      block_bytes_y_b / sizeof(dtype_mma_b);
+
+  static_assert(
+      block_bytes_x_a == block_bytes_y_b,
+      "mat_a x need to match with mat_b y");
+};
+
+/// @brief Specialized for fpu engine.
+template <
+    typename compute_attr_,
+    typename perf_tuning_knob_,
+    typename dtype_scale_,
+    typename dtype_zero_pt_,
+    quant_mode quant_type_,
+    int dequant_s_,
+    mma_engine mma_engine_,
+    gpu_arch arch_tag_>
+struct compute_policy_int4_dequantize<
+    compute_attr_,
+    perf_tuning_knob_,
+    dtype_scale_,
+    dtype_zero_pt_,
+    quant_type_,
+    dequant_s_,
+    mma_engine_,
+    arch_tag_,
+    std::enable_if_t<(
+        arch_tag_ <= gpu_arch::XeHpc && mma_engine_ == mma_engine::fpu)>> {
```

Comment:
looks like `arch_tag_ <= gpu_arch::XeHpc` is not necessaray.

Reply:
yes I will modify